src/test/java/persistence/sql/entity/EntityManagerImplTest.java

```diff
@@ -89,6 +90,20 @@ void entityManager_persist() {
     }
 
 
+    @Test
+    @DisplayName("EntityManager의 persist시 idValue가 null인 경우 id값이 자동으로 생성된다")
+    void entityManager_persist_with_null_id() {
+        long expectedId = 2L;
+        Person expectPerson = new Person(null, "yang2", 25, "rhfpdk92@naver.com");
+
+        entityManager.persist(expectPerson);
+
+        Person resultPerson = entityManager.find(Person.class, expectedId);
```

Comment:
`persist`에 대한 테스트는 해당 메서드의 반환 값을 통해 테스트되면 더욱 좋을 것 같아요 😄 

```suggestion
        Person resultPerson = (Person) entityManager.persist(expectPerson);
```

Reply:
persist에 대한 테스트였는데 작성해둔 테스트는 find로 조회하여 결과를 보여주고있어서 그동안 ID가 null이여도 테스트가 통과했던군요